fhirclient/models/procedure.py

```diff
@@ -194,37 +195,60 @@ def elementProperties(self):
         ])
         return js
 
+    def get_date(self, return_all=False):
+        """
+        if period is None return None
+
+        Return the start date as the default for return_all=False
+        if start is None for some reason, return end
+        if end is also None return None
+        """
+        if self.performedPeriod is None:
+            return None
+        start = self.performedPeriod.start.date if self.performedPeriod.start is not None else None
+        end = self.performedPeriod.end.date if self.performedPeriod.end is not None else None
+
+        if return_all:
+            all_dates = {
+                'start': start,
+                'end': end
+            }
+            return start if start is not None else end, all_dates
```

Comment:
should be:
            return start, all_dates if start is not None else end, all_dates